L1Trigger/L1TMuonEndCap/src/SectorProcessorLUT.cc

```diff
@@ -138,23 +138,55 @@ void SectorProcessorLUT::read(bool pc_lut_data, int pc_lut_version) {
 }
 
 uint32_t SectorProcessorLUT::get_ph_init(int fw_endcap, int fw_sector, int pc_lut_id) const {
-  size_t index = (fw_endcap * 6 + fw_sector) * 61 + pc_lut_id;
-  return ph_init_neighbor_.at(index);
+  const uint32_t index = (fw_endcap * 6 + fw_sector) * 61 + pc_lut_id;
+  uint32_t entry = 0;
+
+  try {
```

Comment:
please avoid try-catch in CMSSW.

Reply:
Please suggest the alternative?

Reply:
if ( index < ph_init_neighbor_.size() )?

Reply:
Ok